protocol.md

```diff
@@ -644,7 +651,9 @@ The "X-" prefix for headers has been deprecated, see [RFC
 
 If you are dealing with HTTP proxies that strip/modify HTTP headers or can't
 handle `PATCH` requests properly, you should consider using HTTPS which will
-make it impossible for proxies to modify your requests.
```

Comment:
> and use the [`Tus-Method-Override`](#tus-method-override) header which allows you to use `POST` requests.

and/or use the [`Tus-Method-Override`](#tus-method-override) header which allows you to use e.g. `POST` requests as substitutes for unsupported methods.